.github/ISSUE_TEMPLATE/alpha_feature.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Alpha Feature Requirements
+about: Requirements for promoting a feature to alpha
+---
+
+This page lists the requirements for promoting a feature to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Requires approved RFC in order to merge. 
+
+**Config**
+
+- [ ] Requires explicit user action to enable (e.g. a config field, config resource, or installation action). 
+
+**Docs**
+
+- [ ] Reference docs are published to istio.io. 
+- [ ] Basic feature docs are published on istio.io describing what the feature does, how to use it, and any caveats. 
+- [ ] Whenever possible, documentation has automated istio.io tests written. 
```

Comment:
Maybe drop the "Whenever possible"

Reply:
isn't automated istio.io tests a beta requirement?  Else, some of our feature will be downgraded.

Reply:
It's listed as an alpha requirement in the definition of done. That said, it could go in either. Users would have to understand that testing hasn't been performed on these alpha features and it may or may not work. I think that's implied though.